app/controllers/rules_controller.rb

```diff
@@ -74,6 +74,7 @@ def update
 
   def destroy
     if @rule.update(deleted_at: Time.zone.now)
+      @rule.additional_answers.destroy_all
```

Comment:
rule also have dependent reviews. was not sure if we want to delete them as well.

Reply:
I think the answer is to delete those as well